### DISCUSSION TITLE
Exporting chart descriptions

### DISCUSSION THREAD

**Discussion by adimyth** [apache/superset#22684] [Ideas]
Is there any reason why superset doesn't export chart descriptions when exporting all assets or charts?

Exporting descriptions would be beneficial, as it would allow reproducibility.

I believe, we can store `description` as another field in the `params` section in exported yaml files.

**Comment by nytai** [comment]
it actually doesn't need to go into params, it can be a regular property. You likely just have to add it here: https://github.com/apache/superset/blob/ad758c080259970c9ea587fd8dd3f56d1dab7722/superset/models/slice.py#L111

want to open a PR?

**Reply by nytai** [reply]
Actually, looks like there's already a PR open to address this https://github.com/apache/superset/pull/22471
